### PR TITLE
Yuqingxie/add beir word piece

### PR DESCRIPTION
**Pull request by amyxie361**
add arguana yaml and template

### CODE REVIEW

File: src/main/resources/docgen/templates/beir-v1.0.0-arguana-wp.template [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Anserini Regressions: BEIR (v1.0.0) &mdash; ArguAna
+
+This page documents BM25 regression experiments for [BEIR (v1.0.0) &mdash; ArguAna](http://beir.ai/).
+These experiments index the corpus in a "flat" manner, by concatenating the "title" and "text" into the "contents" field.
+All the documents and queries are pre-tokenized with bert-base-uncased tokenizer.
```
Comment (lintool) [inline src/main/resources/docgen/templates/beir-v1.0.0-arguana-wp.template:5]:
```suggestion
All the documents and queries are pre-tokenized with `bert-base-uncased` tokenizer.
```

**Review by lintool — APPROVED** [review]
lgtm

**Comment by lintool** [comment]
Also remember to update `README.md`.

**Comment by amyxie361** [comment]
@lintool 
Hi Jimmy, I uploaded the yaml and template for all the datasets, all python script passes the test. Could you double check and I will merge these changes.
One question: do I have to commit the auto-generated .md files?
For README.md, I will add in a separate PR.

Best,
Yuqing

**Comment by lintool** [comment]
Yes, please check in the auto-generated .md files also.

**Pull request merged**